src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = days * 40;
+
+  if (days >= 3 && days < 7) {
+    sum -= 20;
+  } else
+  if (days >= 7) {
+    sum -= 50;
+  }
```

Comment:
avoid else if where it is possible
```suggestion
  }
  
  if (days >= 7) {
    sum -= 50;
  }
```

Reply:
Thank you! corrected